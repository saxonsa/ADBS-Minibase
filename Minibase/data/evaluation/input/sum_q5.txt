Q(z, SUM(1)) :- R(x,y,z)