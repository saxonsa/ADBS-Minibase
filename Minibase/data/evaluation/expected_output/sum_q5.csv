'adbs', 1
'anlp', 1
'ids', 1
'sdp', 1
'mlpr', 1
'rl', 1
'ppls', 1
